blog/r-tidyfinance-0-1-0/index.qmd

```diff
@@ -0,0 +1,185 @@
+
+In this blog post, we demonstrate the features of the initial release. We decided to focus on functions that allow you to download the data that we use in TFWR. 
+
+## Install the package
+
+You can install the released version of `tidyfinance` from CRAN via:
+
+```{r}
+#| eval: false
+install.packages("tidyfinance")
+```
+
+You can install the development version of tidyfinance from GitHub using the `pak` package:
+
+```{r}
+#| eval: false
+pak::pak("tidy-finance/r-tidyfinance")
+```
+
+## Download data 
+
+Let's start by loading the package
+
+```{r}
+library(tidyfinance)
+```
+
+The main function is `download_data(type, start_date, end_date)` with supported type:
+```{r}
+list_supported_types()
+```
+
+So, for instance, if you want to download monthly Fama-French Three-Factor data, you can call:
+```{r}
+#| message: false
+download_data("factors_ff3_monthly", "2020-01-01", "2020-12-31")
+```
+Under the hood, the function uses the `frenchdata` package and applies some cleaning steps, as in TFWR. If you haven't installed `frenchdata` yet, you'll get prompted to install it first before you can download this specific data type. 
+
+You can also access q-Factor data in this way, by calling:
+
+```{r}
+download_data("factors_q5_daily", "2020-01-01", "2020-12-31")
+```
+
+To ensure that we can extend the functionality of the download functions for specific types, we also provide domain-specific download functions. The `download_data("factors_ff3_monthly")` actually calls `download_data_factors("factors_ff3_monthly")`, which in turn calls `download_data_factors_ff("factors_ff3_monthly")`. Why did we decide to have these nested function approach?
+
+Suppose that the q-Factor data changes its URL path and our original function does not work anymore. In this case, you can replace the default `url` value in `download_data_factors_q(type, start_date, end_date, url)` to apply the usual cleaning steps. 
+
+This feature becomes more apparent for other data sources such as `wrds_crsp_monthly`. Note that you need to have valid WRDS credentials and need to set them correctly (check `?get_wrds_connection` and [WRDS, CRSP, and Compustat](../../r/wrds-crsp-and-compustat) in TFWR). If you want to download the standard monthly CRSP data, you can call:
+
+```{r}
+#| output: false
+#| eval: false
+download_data("wrds_crsp_monthly", "2020-01-01", "2020-12-31")
+```
+
+However, if you want to add further columns, you can add them via `...` to `download_data_wrds_crsp()`, for instance:
+
+```{r}
+#| output: false
+#| eval: false
+download_data_wrds_crsp("wrds_crsp_monthly", "2020-01-01", "2020-12-31", mthvol)
+```
+
+Similarly, if you want to download the old version of CRSP, you can use the `version` parameter in `download_data_wrds_crsp()` (see this blog post here TODO: INSERT LINK). 
+
```

Comment:
Maybe copy one sentence on what "old" means and clarify that the data is downloaded using CRSP 2.0